### PR TITLE
Ctm test for testing CTM/CPR functionality.

### PR DESCRIPTION
**Pull request by jsarha**

### CODE REVIEW

File: py/tests/ctm_csc.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+crtc.set_prop("CTM", ctmb.id)
+
+input("press enter to set r->g g->b b->r ctm\n")
+
+ctm = [ 0.0,	1.0,	0.0,
+        0.0,	0.0,	1.0,
+        1.0,	0.0,	0.0 ]
+
+ctmb = ctm_to_blob(ctm, card)
```
Comment (tomba) [inline py/tests/ctm_csc.py]: Here you overwrite the variable containing the previous blob, which causes the previous blob to be destroyed and, I guess, will result in a modeset which disables the CTM from the HW (correct me if I'm wrong). That's not a big issue, but maybe the test should keep the old blob alive until the new one has been set.

Comment (jsarha) [inline py/tests/ctm_csc.py]: But the previous blob has been committed already. Surely, and also according to result of my brief test [1], kernel side keeps a reference of blobs that are part of the current state. I would call any other behaviour a bug.

I added a following sequence:
-----------------------------------------------------------------
ctm = [ 0.0,	1.0,	0.0,
        0.0,	0.0,	1.0,
        1.0,	0.0,	0.0 ]

ctmb = ctm_to_blob(ctm, card)

crtc.set_prop("CTM", ctmb.id)

input("test")

ctmb = 0

input("test")

crtc.set_mode(conn, fb, mode)

input("test")
-----------------------------------------------------------------
... and the committed color matrix remains active also after setting the crtc mode.

Comment (tomba) [inline py/tests/ctm_csc.py]: Maybe... But at least if you destroy an fb, it'll cause a modeset and removal of the fb from the display. And fbs are similar to blobs in many ways, so I thought the same applies to blobs. But I admit that I have never actually tested it.

**Comment by jsarha** [comment]
I update the CTM matrix filling to be in sign + magnitude format.

**Comment by jsarha** [comment]
This is an obsolete request.

**Pull request closed without merging**